src/pages/index.js

```diff
@@ -104,35 +129,107 @@ function closeModalByEscape(event) {
   }
 }
 
+// Declare and cache DOM elements at the top
+const saveButtonEdit = editForm.querySelector(".modal__submit-btn");
+const saveButtonCard = cardForm.querySelector(".modal__submit-btn");
+const saveButtonAvatar = avatarForm.querySelector(".modal__submit-btn");
+
+//
+
+// Function to reset button text and state
+function resetSaveButton(saveButton) {
+  saveButton.textContent = "Save";
+  // saveButton.disabled = false;
+  // saveButton.classList.remove("modal__submit-btn_disabled");
+}
+
 // Handle Edit Form Submit
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = profileNameInput.value;
-  profileDescription.textContent = profileDescriptionInput.value;
-  closeModal(editModal);
+  saveButtonEdit.textContent = "Saving...";
+
+  api
+    .editUserInfo({
+      name: profileNameInput.value,
+      about: profileDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      toggleButtonState(
+        [profileNameInput, profileDescription],
+        saveButtonEdit,
+        validationConfig
```

Comment:
`toggleButtonState` is not needed when you submit the profile form because it doesn't have any effect (you don't reset the form before `toggleButtonState`)